pkg/testrailuttils/testrailutils.go

```diff
@@ -0,0 +1,199 @@
+package testrailuttils
+
+import (
+	"fmt"
+	"github.com/educlos/testrail"
+	"github.com/portworx/torpedo/drivers/volume"
+	"github.com/sirupsen/logrus"
+)
+var (
+	milestoneId int
+	client      *testrail.Client
+	runId       int
+	testRailConnectionSuccessful bool
+	MilestoneName string
+	RunName string
+	JobRunId string
+	JenkinsBuildUrl string
+)
+const(
+	pwxProjectId                          = 1
+	testRailPassStatusCode                = 1
+	testRailFailStatusCode                = 5
+)
+
+type Testrail struct{
+	Status string
+	TestId int
+	RunId int
+	V volume.Driver
```

Comment:
You don't need to include the entire volume driver here. Add VolumeVersion and get it from outside of this struct